docker-compose/dse-next/docker-compose-dev-mode.yml

```diff
@@ -0,0 +1,112 @@
+version: '2'
+
+services:
+  coordinator:
+    image: stargateio/coordinator-dse-next:${SGTAG}
+    networks:
+      - stargate
+    ports:
+      - "9042:9042"
+      - "8081:8081"
+      - "8090:8090"
+    mem_limit: 2G
+    cap_add:
+      - IPC_LOCK
+    ulimits:
+      memlock: -1
+    environment:
+      - JAVA_OPTS="-Xmx1536M"
+      - CLUSTER_NAME=c5-${DSETAG}-cluster
+      - RACK_NAME=rack1
+      - DATACENTER_NAME=datacenter1
+      - ENABLE_AUTH=true
+      - DEVELOPER_MODE=true
+    healthcheck:
+      test: curl -f http://localhost:8084/checker/readiness || exit 1
+      interval: 15s
+      timeout: 10s
+      retries: 10
+
+  restapi:
+    image: stargateio/restapi:${SGTAG}
+    depends_on:
+      coordinator:
+        condition: service_healthy
+    networks:
+      - stargate
+    ports:
+      - "8082:8082"
+    mem_limit: 2G
+    environment:
+      - JAVA_MAX_MEM_RATIO=75
+      - JAVA_INITIAL_MEM_RATIO=50
+      - GC_CONTAINER_OPTIONS=-XX:+UseG1GC
+      - QUARKUS_HTTP_CORS=true
+      - QUARKUS_HTTP_CORS_ORIGINS=*
+      - QUARKUS_HTTP_CORS_METHODS=POST,GET,PUT,OPTIONS,DELETE
```

Comment:
Do we need CORS support?

Reply:
This is copied from other compose file. These flags were added by Jake to get the REST `/cql` endpoint working.

Reply:
Yeah Got it. This is a separate discussion then.